packages/mui-base/src/useSlider/useSlider.types.ts

```diff
@@ -69,6 +69,11 @@ export interface UseSliderParameters {
    * @default 'horizontal'
    */
   orientation?: 'horizontal' | 'vertical';
+  /**
+   * The granularity with which the slider can step through values when using Page Up/Page Down or Shift + Arrow Up/Arrow Down.
+   * @default 10
+   */
+  pageStep?: number;
```

Comment:
A similar prop in the NumberInput is called `shiftMultiplier`. We should agree on a common name and use it in both components

Reply:
As a name `shiftMultiplier` makes sense, I would go with updating the PR. I came up with this name because we already had the `step` prop. @colmtuite any preference on the name?

Reply:
The name wouldn't be the only thing changing, though: `shiftMultiplier` multiplies the step, and `pageStep` is the multiplication result. I agree that we should follow one pattern in both (and all) components.

Reply:
Yeah `multiplier` as part of the name may be misleading. Maybe `shiftStep` should be an alternative option considering the same action is happening both on Page Up/Down and Shift + Arrow Up/Down - maybe "shift" would be better as the slider does not have a notion of "pages" as other components may have.

Reply:
Yeah `shiftStep` could work too.

> "Page Up (Optional): Increase the slider value by an amount larger than the step change made by Up Arrow."

Aria docs does suggest that Page Up/Down would increment/decrement the slider by a larger step. So I think `pageStep` applies to both components.

Aria docs mentions "large step" or "larger step" a few times, which I guess would translate to `largeStep`. So that could be an option.

So I guess `pageStep`, `shiftStep`, and `largeStep` are all viable options. I think my I'd put them in that order of preference.

Reply:
No one shared their opinion, so I guess we can then keep `pageStep`. I'll keep the PR open for a day to allow people to respond, then will merge it :)

Reply:
I like "largeStep" better than "pageStep", since there is no concept of pages in a slider. It's also pretty self-explanatory.

Reply:
> Maybe `shiftStep` should be an alternative option considering the same action is happening both on Page Up/Down and Shift + Arrow Up/Down - maybe "shift" would be better as the slider does not have a notion of "pages" as other components may have.

I still hold on to this thought ☝️ It means both shifting, and it indicates you can use the `Shift` key :)) But honestly I don't think any of them is a bad name, I think it's all personal preference.

@colmtuite or @michaldudak would we even started this discussion if that was the original name? 😄 If not, I think we should use it ;)

Reply:
I'm happy with either of them, I don't think any of them are problematic. They all work fine imo. So since you're doing to PR, I give a vote to you for `shiftStep` 😄

Reply:
Yup, I'm not against any of these either. Make it `shiftStep`, then :)

Reply:
Alright, applying the changes.